manticore/core/smtlib/visitors.py

```diff
@@ -147,6 +147,7 @@ def visit_Operation(self, expression, *operands):
         else:
             self._print('...')
         self.indent -= 2
+        return ''
```

Comment:
Yes but this has to be applied only for the pretty printer. The optimization needs `_method` to iterate over all the `visit_` methods. But we can define a particular `_method` for the pretty printer with this behavior